opentracing/src/main/java/opentracing/Tracer.java

```diff
@@ -31,7 +36,9 @@
    *                   .setTag("lucky_number", 42);
    * }</pre>
    */
-  Span startTrace(String operationName);
+  Span startSpan(String operationName, Optional<Span> parent);
+
+  <T> Injector<T> getInjector(Class<T> carrierType);
```

Comment:
@tschottdorf morally speaking, yes. Though of course the naming is different and they're both ultimately methods on Tracer.

The biggest difference is more a matter of convention than a matter of the literal interface, at least in Go: the "required carriers" would include types like `opentracing.SplitBinaryCarrier` and so forth: i.e., definitely not `map[string]string`. `http.Header` might be ok, though that's on the border. In dynamically-typed languages there might still be the need for something like the current API's `format`.

Anyway, as much as I like sweeping Injector and Extractor under the rug (since most tracing instrumentation _eventually_ is intra-process rather than inter-process), the 1.0RC API is arguably too abstract. Curious to hear what others think.

(I realize it's potentially annoying to vacillate about this, but better to adjust things now than later if we come to consensus about a better path)